component/componentattribute/attribute.go

```diff
@@ -0,0 +1,12 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package componentattribute // import "go.opentelemetry.io/collector/component/componentattribute"
+
+const (
+	ComponentKindKey = "otelcol.component.kind"
+	ComponentIDKey   = "otelcol.component.id"
+	PipelineIDKey    = "otelcol.pipeline.id"
+	SignalKey        = "otelcol.signal"
+	SignalOutputKey  = "otelcol.signal.output"
```

Comment:
I would not couple this change with breaking the keys.
```
	zapKindKey            = "kind"
	zapNameKey            = "name"
	zapDataTypeKey        = "data_type"
	zapStabilityKey       = "stability"
	zapPipelineKey        = "pipeline"
	zapExporterInPipeline = "exporter_in_pipeline"
	zapReceiverInPipeline = "receiver_in_pipeline"
```

Reply:
I'm not sure what you're suggesting